WTWSMS/decisions/LI_target_decisions.txt

```diff
@@ -1,4 +1,4 @@
-targetted_decisions = {	
+decisions = {	
```

Comment:
Or, we could just make "CONVERT TO ATTACKER'S RELIGION" a proper Decision, like it should be.